src/object-store/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ include(CompilerFlags)
 include(Sanitizers)
 
 include(RealmCore)
-set(REALM_CORE_VERSION "1.1.0" CACHE STRING "")
+set(REALM_CORE_VERSION "1.2.0" CACHE STRING "")
```

Comment:
This might explain the test failures. Object store no longer has the workaround for realm/realm-core#1837, as it was fixed in core v1.2.0.